misc/kickstart_create_network_line.py

```diff
@@ -231,43 +238,46 @@ def useable_interfaces(net_devs, nc, iface_speed):
     dev_list = bondable_iface
     dev_str = dev_list.pop()
     for d in dev_list:
-        dev_str="%(dev_str)s,%(d)s" %vars()
+        dev_str=",".join([dev_str, d])
 else:
     dev_str = bondable_iface[0]
 
 
 if ('y' in nc['NETWORKING_IPV6'].lower()) and re.search(":",nc['IPV6ADDR']):
-    temp_var = nc['IPV6ADDR']
-    IPV6 = "--ipv6=%(temp_var)s" %vars()
+    IPV6 = "".join(["--ipv6=", nc["IPV6ADDR"]])
 else:
     if 'y' in nc['NETWORKING_IPV6'].lower():
-        if iface_problems is False: 
+        if iface_problems is False:
             iface_problems = "IPv6 enabled but no address provided"
-        else: 
+        else:
             iface_problems = "{0} and IPv6 enabled but no address provided".format(iface_problems)
     if re.search(":",nc['IPV6ADDR']):
         if iface_problems is False:
             iface_problems = "IPv6 is disabled, but IPV6ADDR was set to {0}".format(nc['IPV6ADDR'])
         else:
-            iface_problems = "{0} and IPv6 is disabled, but IPV6ADDR was set to {1}".format(iface_problems,nc['IPV6ADDR'])
+            iface_problems = " ".join([iface_problems, "and IPv6 is disabled, but IPV6ADDR was set to", nc['IPV6ADDR']])
```

Comment:
I would leave the original line as-is

Reply:
This again just came down to string concatenation - I wanted to be consistent. What about, instead of the original line, this was made to use `+` as above?

Reply:
I don't think _that_ will work here because `iface_problems` appears to be a bool

Reply:
Well I could just cast it to a string.

Reply:
Actually, `format`'s way better in that case. I'll just change it back.